Dockerfile

```diff
@@ -41,12 +41,11 @@ COPY schema ./schema
 COPY scripts ./scripts
 COPY tools ./tools
 COPY vast ./vast
-COPY .clang-format .cmake-format LICENSE LICENSE.3rdparty README.md BANNER CHANGELOG.md CMakeLists.txt configure vast.yaml.example ./
-RUN ./configure \
-    --prefix=$PREFIX \
-    --build-type=$BUILD_TYPE \
-    --log-level=INFO \
-    --generator=Ninja
+COPY .clang-format .cmake-format LICENSE LICENSE.3rdparty README.md BANNER CHANGELOG.md CMakeLists.txt vast.yaml.example ./
+RUN cmake -B build -G Ninja \
+    -DCMAKE_INSTALL_PREFIX="$PREFIX" \
```

Comment:
Considering there's no need for this to be relocatable inside the Docker container we might want to consider enabling `VAST_ENABLE_RELOCATABLE_INSTALLATIONS:BOOL=OFF`.